Scripts/RemediationScripts/Remediate-EnableHTTPSForAppServices.ps1

```diff
@@ -216,18 +221,9 @@ function Enable-HttpsForAppServices
     Write-Host "Account Type: $($context.Account.Type)"
     Write-Host $([Constants]::SingleDashLine)
 
-    Write-Host "Checking if $($context.Account.Id) is allowed to run this script..."
-
-    # Checking if the current account type is "User"
-    if ($context.Account.Type -ne "User")
-    {
-        Write-Host "WARNING: This script can only be run by `User` Account Type. Account Type of $($context.Account.Id) is: $($context.Account.Type)" -ForegroundColor $([Constants]::MessageType.Warning)
-        break
-    }
-
     Write-Host "*** To enable HTTPS for App Services in a Subscription, Contributor and higher privileges on the App Services are required. ***" -ForegroundColor $([Constants]::MessageType.Info)
```

Comment:
In this script we don't have check for permission like minimum Contributor?

Reply:
No, we do not check for each level (resource/RG/subscription).
We are just giving a warning message.
If user doesn't have permission, Set command will throw an error.